fog/ocall_oram_storage/trusted/src/lib.rs

```diff
@@ -94,12 +98,12 @@ lazy_static! {
     static ref OCALL_REENTRANCY_MUTEX: Mutex<()> = Mutex::new(());
 }
 
-/// Cipher type. Anything implementing StreamCipher and NewCipher at 128
+/// Cipher type. Anything implementing StreamCipher and KeyIvInit at 128
 /// bit security should be acceptable
-type CipherType = Aes256Ctr;
+type CipherType = Ctr64BE<Aes256>;
```

Comment:
See https://docs.rs/aes/0.7.5/src/aes/autodetect.rs.html#147 for the original definition of `Aes256Ctr`